DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -57,118 +57,122 @@
           </a>
         </div>
 
-        <div class="nhsuk-header__content" id="content-header">
-          <div class="nhsuk-header__menu">
-            <button class="nhsuk-header__menu-toggle" id="toggle-menu" aria-controls="header-navigation" aria-label="Open menu">Menu</button>
+        @if (ViewData["DoNotDisplayNavBar"] == null) {

```

Comment:
It might be nicer to check != true to handle DoNotDisplayNavBar being false case :)